frontend/src/concepts/k8s/utils.ts

```diff
@@ -12,5 +12,14 @@ export const translateDisplayNameForK8s = (name: string): string =>
     .replace(/\s/g, '-')
     .replace(/[^A-Za-z0-9-]/g, '');
 
+export const translateNameForWb = (name: string): string => {
```

Comment:
I like the way you did this -- but I think we're still going to cause an issue with `translateDisplayNameForK8s` on other resources -- can we not just take an optional prefix or something that makes the signature:

```
/** 
 * Converts a display name to a k8s safe variant, if needed a `safeK8sPrefix` can be provided to customize the prefix used to align with k8s standards if it is needed.
 */
export const translateDisplayNameForK8s = (name: string, safeK8sPrefix?: string = 'ai-'): string
```
And we can write the logic for checking if it is valid to add the prefix as needed?

I don't imagine we want to inject a common pattern always, so like your if-statement below, we should have that condition still. 

And you can provide `wb-` for the workbench resource.

Reply:
Good idea! Updated accordingly.